packages/framework-provider-azure/src/library/api-adapter.ts

```diff
@@ -25,6 +34,7 @@ export async function requestSucceeded(
     },
     status: 200,
     body: body ? JSON.stringify(body) : '',
+    ...extraParams,
```

Comment:
Shouldn't the `Subprotocol` web socket header go inside the "headers" key? Is this something special to Azure web sockets?

Reply:
It's in the header, but in Azure we need it in the body too